apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/pathAssertions.kt

```diff
@@ -0,0 +1,297 @@
+@file:Suppress("JAVA_MODULE_DOES_NOT_READ_UNNAMED_MODULE" /* TODO remove once https://youtrack.jetbrains.com/issue/KT-35343 is fixed */)
+
+package ch.tutteli.atrium.api.infix.en_GB
+
+import ch.tutteli.atrium.api.infix.en_GB.creating.path.PathWithCreator
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.builders.path
+import java.nio.file.Path
+
+/**
+ * Expects that the subject of the assertion (a [Path]) starts with the [expected] [Path].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.startsWith(expected: Path): Expect<T> =
+    addAssertion(ExpectImpl.path.startsWith(this, expected))
+
+/**
+ * Expects that the subject of the assertion (a [Path]) does not start with the [expected] [Path].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.startsNotWith(expected: Path): Expect<T> =
+    addAssertion(ExpectImpl.path.startsNotWith(this, expected))
+
+/**
+ * Expects that the subject of the assertion (a [Path]) ends with the expected [Path].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.endsWith(expected: Path): Expect<T> =
+    addAssertion(ExpectImpl.path.endsWith(this, expected))
+
+/**
+ * Expects that the subject of the assertion (a [Path]) does not end with the expected [Path];
+ *
+ * @param expected The [Path] provided to the assertion
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.endsNotWith(expected: Path): Expect<T> =
+    addAssertion(ExpectImpl.path.endsNotWith(this, expected))
+
+/**
+ * Expects that the subject of the assertion (a [Path]) exists;
+ * meaning that there is a file system entry at the location the [Path] points to.
+ *
+ * This matcher _resolves_ symbolic links. Therefore, if a symbolic link exists at the location the subject points to,
+ * then the search will continue at that location.
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.12.0
+ */
+infix fun <T : Path> Expect<T>.to(@Suppress("UNUSED_PARAMETER") exist: exist): Expect<T> =
```

Comment:
the problem of the build is this function, I always figured we might run into problems and tada, now we do. 
The good thing, since it is a new function in this API we can rename it -- you need to adjust the ReplaceWith in the jdk8 extension accordingly though. I suggest we use:
```
toBe(existing)
```